README.md

```diff
@@ -22,15 +24,14 @@ The Git repository with the method **must** have the following files
 
 Additionally, the Git repository **must** also have the [necessary files for setting up a binder environment](https://mybinder.readthedocs.io/en/latest/using/config_files.html)[^1]. For example,
 
-- `binder/postBuild`
+- `binder/postBuild` from [postBuild](https://methodshub.gesis.org/snippet/postBuild)
```

Comment:
Could we say that this is just an example? I don't think that `postBuild` would fit all situations.